packages/@aws-cdk/aws-sqs/lib/queue.ts

```diff
@@ -245,9 +251,12 @@ export class Queue extends QueueBase {
 
     const { encryptionMasterKey, encryptionProps } = _determineEncryptionProps.call(this);
 
+    const fifoProps = this.determineFifoProps(props);
+    this.fifo = (typeof(fifoProps.fifoQueue) !== 'undefined' && fifoProps) ? true : false;
```

Comment:
style:

```ts
this.fifo = fifoProps && fifoProps.fifoQueue;
```

Reply:
This exact suggestion is not going to work - when `fifoProps` is `undefined`, `this.fifo` also ends up `undefined`.

However, I've corrected this line to be of the style you're suggesting.